Book/tests/running_the_test_suite.rst

```diff
@@ -56,7 +56,8 @@ You can also specify multiple target folders or files to run.
 
     ~/php-src> sapi/cli/php run-tests.php -P Zend/ ext/reflection/ ext/standard/tests/array/
 
-Hopefully all the tests will have passed. If not... oops.
+Hopefully all the tests will have passed. If not, the script will ask if you want to send a report to the PHP QA team.
+This can be annoying sometimes. It is possible suppress this question by setting ``NO_INTERACTION=1``.
```

Comment:
I would also mention `-q`, which does the same thing.

Reply:
Thanks! TIL :-)

I have more things in progress but didn't have time yet to make it ready for PR. I hope to do this any day now.